<launch>
  <!--  smooths inputs from cmd_vel_mux/input/teleop_raw to cmd_vel_mux/input/teleop -->
  <include file="$(find turtlebot_teleop)/launch/includes/velocity_smoother.launch.xml"/>

  <node pkg="turtlebot_teleop" type="ps4_joy" name="ps4_joy">
    <param name="scale_angular" value="1.0"/>
    <param name="scale_linear" value="0.8"/>
    <remap from="turtlebot_teleop_joystick/cmd_vel" to="teleop_velocity_smoother/raw_cmd_vel"/>
  </node>
  
  <node pkg="joy" type="joy_node" name="joy_node">
  </node>

</launch>

